# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/top-level-kmod.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Having all all in-kernel modules in one top-level kernel/ directory
is a bit superflous. Let's just install all sub-systems into the
top-level directory to shorten all the pathes for readability.

--- linux-6.1/scripts/Makefile.modinst.vanilla	2023-02-06 00:25:41.525322193 +0100
+++ linux-6.1/scripts/Makefile.modinst	2023-02-06 00:25:44.378321927 +0100
@@ -12,7 +12,7 @@
 modules := $(sort $(shell cat $(MODORDER)))
 
 ifeq ($(KBUILD_EXTMOD),)
-dst := $(MODLIB)/kernel
+dst := $(MODLIB)
 else
 INSTALL_MOD_DIR ?= extra
 dst := $(MODLIB)/$(INSTALL_MOD_DIR)
